pkg/prom/cleaner_test.go

```diff
@@ -0,0 +1,170 @@
+package prom
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/grafana/agent/pkg/prom/instance"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Basic logger to write to stderr to help debug test failures
+func getLogger() log.Logger {
+	return log.NewLogfmtLogger(os.Stderr)
+}
+
+func TestWALCleaner_getAllStorageNoRoot(t *testing.T) {
+	walRoot := filepath.Join(os.TempDir(), "getAllStorageNoRoot")
+	cleaner := NewWALCleaner(getLogger(), &mockManager{}, walRoot)
+
+	// Bogus WAL root that doesn't exist. Method should return no results
+	wals := cleaner.getAllStorage()
+
+	assert.Empty(t, wals)
```

Comment:
It's a little odd to see `require` and `assert` mixed together; can we consistently use `require` throughout the tests?

Reply:
Whoops, will fix.